src/pages/index.js

```diff
@@ -49,35 +65,115 @@ function handleImageClick(data) {
 }
 
 function handleEditSumbit(obj) {
+  popupEdit.renderLoading(true);
   userInfo.setUserInfo(obj);
-  popupEdit.close();
-  formEditValidator.disableButton();
+  api
+    .updateCurrentUser(
+      JSON.stringify({
+        name: obj.name,
+        about: obj.about,
+      })
+    )
+    .then(() => {
+      popupEdit.close();
+      popupEdit.renderLoading(false);
+      formEditValidator.disableButton();
+    });
 }
 
 function handleAddSumbit(obj) {
-  sectionCard.addItem(createCard(obj));
-  popupAdd.close();
-  popupAdd.form.reset();
-  formAddValidator.disableButton();
+  popupAdd.renderLoading(true);
+  api
+    .postCard(
+      JSON.stringify({
+        name: obj.name,
+        link: obj.link,
+      })
+    )
+    .then((data) => {
+      sectionCard.addItem(data, "prepend");
+    })
+    .then(() => {
+      setTimeout(10000);
```

Comment:
`setTimeout(10000)` is not needed here. Everything should work **naturally** by waiting the response from the server